src/common/sha2/sha2_armv8.c

```diff
@@ -214,6 +234,7 @@ void oqs_sha2_sha256_inc_finalize_armv8(uint8_t *out, sha256ctx *state, const ui
 		out[i] = state->ctx[i];
 	}
 	oqs_sha2_sha256_inc_ctx_release_c(state);
+	free(tmp_in); // IGNORE free-check
```

Comment:
I think it would be better to use `OQS_MEM_secure_free` here because it's possible in principle that we are hashing secret data.

Reply:
Done.